### ISSUE TITLE
Support for more than just textarea?

### ISSUE THREAD

**Issue by antonywu** [tiff/wysihtml5#73]
With Aloha, it is possible to turn practically any elements into an editing area, to achieve the illusion of "WYSIWYG" effect.
Furthermore, does wysihtml5 currently allow for editing in multiple fields in the same page? 

I admit that I am practically asking for features that we see in Aloha, but Aloha is still huge, so perhaps, wysihtml5 can take on Aloha and stay slim and fast.

Thanks for reading.

**Comment by arjun-urs** [comment]
+1. multiple text fields per page would be a great feature

**Comment by nevf** [comment]
@arjun-urs what is your specific problem as I thought it did work fine with multiple textarea's. The only thing that may be an issue is where you only want a single toolbar. I plan to look into this shortly.

**Comment by arjun-urs** [comment]
@nevf : I mean multiple text fields with a single toolbar.

**Comment by nevf** [comment]
@arjun-urs Yes that's what I need as well. See my post https://github.com/xing/wysihtml5/issues/32 

I finally made time to work on this yesterday, and have the basic changes to the Toolbar code in place. So far I have only found one problem. If you'd like a copy let me know.

**Comment by r043v** [comment]
i done that too, just using one editor, one toolbar and multiple source div, just populate the textarea on live with div content, place the area on the exact div place, resize ifrm if need and here we go !

i'll will release my code when it will be really clean

edit > alpha released [here](https://github.com/r043v/jQuery.scribe)

**Comment by everlaat** [comment]
+1 to! Would be a gret feature, its the reason i sticked to nicedit for a long time

**Comment by bpiero** [comment]
+1
